neural_speed/convert/__init__.py

```diff
@@ -21,13 +21,12 @@
 model_maps = {"gpt_neox": "gptneox", "gpt_bigcode": "starcoder", "whisper": "whisper", "qwen2": "qwen"}
 
 
-def convert_model(model, outfile, outtype="f32", model_hub="huggingface", use_quantized_model=False):
+def convert_model(model, outfile, outtype="f32", format="NE", model_hub="huggingface", use_quantized_model=False):
```

Comment:
why add new args?

Reply:
@a32543254 During the conversion there are 2 ways either to NE format or GGUF, but we don't give to the user the choice of the format. Exposing this argument, allows the user to choose which format to use during the conversion.

Reply:
Yes, not all models support gguf for now, so in this api, we only support ne now.
When support all models with gguf could be supported , then we can add this args.

Reply:
have to change this probably. otherwise cause other models conversion fail maybe.

Reply:
@Zhenzhong1 It's working also for other models, you can try. It's an optional argument, by default it will take ne format, and I add --format only for Phi and Stablelm that support gguf ;-)

Reply:
nicely done!!!